pkgs/applications/misc/hyperspace/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, fetchurl, lib
+, autoPatchelfHook, wrapGAppsHook, dpkg
+, libX11, libXext, libXi, libXau, libXrender, libXft, libXmu, libSM, libXcomposite, libXfixes, libXpm
+, libXinerama, libXdamage, libICE, libXtst, libXaw, fontconfig, pango, cairo, glib, libxml2, atk, gtk3
+, gdk-pixbuf
+# more
+, nss, nspr
+, libXScrnSaver
+, alsaLib
+# rt?
+, udev, libGL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hyperspace";
+  version = "1.0.3";
+  src = fetchurl {
+    url = "https://github.com/hyperspacedev/${pname}/releases/download/v${version}/${pname}_${version}_amd64.deb";
+    sha256 = "1vain39hhw6zb854l4f62ac262i2vy1pjw557ggmd8yhh2l94vas";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    #wrapGAppsHook
+    dpkg
+  ];
+
+  unpackPhase = ''
+    dpkg-deb -vx $src .
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+
+    # /usr/share
+    mv ./usr/* $out/
+
+    # /opt/Hyperspace\ Desktop
+    mv ./opt/Hyperspace\ Desktop $out/share/hyperspace
+
+    mkdir -p $out/bin
+    ln -s $out/share/hyperspace/hyperspace $out/bin/hyperspace
+
+    rm -vrf $out/share/hyperspace/{libGLESv2.so,libEGL.so,swiftshader}
+
+    substituteInPlace $out/share/applications/hyperspace.desktop \
+     --replace 'Exec="/opt/Hyperspace Desktop/hyperspace"' \
+               'Exec="$out/bin/hyperspace"'
+  '';
+
+  buildInputs = [
+    # From maxx, only w/gtk3
+    stdenv.cc.cc libX11 libXext libXi libXau libXrender libXft libXmu libSM libXcomposite libXfixes libXpm
+    libXinerama libXdamage libICE libXtst libXaw fontconfig pango cairo glib libxml2 atk gtk3
+    gdk-pixbuf
+    # more
+    nss nspr
+    libXScrnSaver
+    alsaLib
+  ];
+
+  runtimeDependencies = [ libGL udev.lib ];
```

Comment:
didn't know about this, nice :)

Reply:
Apparently it's documented but I didn't either! Found it in other expressions when trying to fix some issue while packaging haha :).